_posts/2020-09-17-using-tettra.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "Using Tettra to Keep our Content Organized"
+date: 2020-09-17 11:00:00
+reviewed: 2020-09-17 11:00:00
+categories: ["team", "remote"]
+author: "abizzinotto"
+---
+
+At OmbuLabs we use [Tettra](https://tettra.com/) to power our knowledge base. As a remote, distributed team it is very important to us to have our content structured and organized. We follow some good practices to keep it organized and up to date, which we describe [here](https://www.ombulabs.com/blog/team/agile/remote/organizing-team-knowledge-bases.html), and we use Tettra to keep our knowledge organized and integrated with the tools we use on a daily basis.
```

Comment:
@ABizzinotto Maybe you can explain a little bit more about our motivation behind this? I think many remote teams can relate to this: Slack is a great communication tool but it is not a great knowledge base. We kept having important conversations and decisions made in Slack, but then it was very hard to go back and search through Slack's history to find threads that were key to internal and client projects.

Reply:
@etagwerker the idea is for this article to be about the tool itself. The motivation and best practices on a knowledge base where shared on this other article here https://www.ombulabs.com/blog/team/agile/remote/organizing-team-knowledge-bases.html

So I didn't want to be repetitive here.